setup.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
+
 import provider
 
 setup(
-    name='django-oauth2-provider',
+    name='edx-django-oauth2-provider',
```

Comment:
To me, this seems like just punting on the problem.  Won't every developer have to deal with this somehow? Or worse, the code will kind of sort of work and we won't figure out until later that the problem with Joe's devstack is that he still has the wrong package installed?
If you update this: https://github.com/edx/edx-platform/blob/master/pavelib/prereqs.py#L173 then edx-platform will do the uninstall for you.